dist/stat_data.py

```diff
@@ -528,6 +528,9 @@ def __init__(self, name, desc, flags=''):
     RecStat('rec_page_delete_fast', 'fast-path pages deleted'),
     RecStat('rec_pages', 'page reconciliation calls'),
     RecStat('rec_pages_eviction', 'page reconciliation calls for eviction'),
+    RecStat('rec_pages_with_txn', 'page reconciliation calls that resulted in pages with transaction ids'),
```

Comment:
The stats need to be in the alphabetic order.

Reply:
Fixed